src/test/java/pl/java/exercises/day1/task11/Circle_11_Test.java

```diff
@@ -0,0 +1,49 @@
+package pl.java.exercises.day1.task11;
+
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.within;
+
+class Circle_11_Test {
+    private Circle circle;
+    @BeforeEach
+    void setUp() {
+        Point2D center = new Point2D(0, 0);
+        Point2D point = new Point2D(3, 0);
+        circle = new Circle(center, point);
+    }
+    @Test
+    void shouldReturnResizedRadius(){
+        //given
+        //when
+        double result = circle.resize(2);
```

Comment:
z tego co widzę to resize(2) występuje w każdym teście, więc można rozważyć przeniesienie tego do metody setUp